base/db/tests/testthat.R

```diff
@@ -10,7 +10,12 @@ library(testthat)
 
 library(PEcAn.DB)
 library(RPostgreSQL)
-dbparms <- list(host = "localhost", driver = "PostgreSQL", user = "bety", dbname = "bety", password = "bety")
+dbparms <- get_postgres_envvars(
```

Comment:
would it make sense to have most of these coded in get_postgres_envvars as defaults?

Reply:
I wondered that too, and could be persuaded either way. At the moment I leaning toward thinking it's less magic without, because then what you get back is strictly what you put in modulo whatever PG* vars are set. Thoughts?

Reply:
roll a 🎲 and see what happens. I can be convinced of either solution, hence I approved the pull request.